packages/functional-tests/tests/react-conversion/oauthPromptNone.spec.ts

```diff
@@ -0,0 +1,243 @@
+      await signin.fillOutEmailFirstForm(credentials.email);
+      await signin.fillOutPasswordForm(credentials.password);
+
+      //Verify sign up code header
+      await expect(page).toHaveURL(/confirm_signup_code/);
+
+      const query = new URLSearchParams({
+        login_hint: credentials.email,
+        return_on_error: 'false',
+      });
+      await page.goto(`${target.relierUrl}/?${query.toString()}`);
+      await relier.signInPromptNone();
+
+      //Verify error message
+      await expect(page.getByText('Unverified user or session')).toBeVisible();
+    });
+  });
+
+  test.describe('oauth prompt none with emails', () => {
+    test('succeeds if login_hint same as logged in user', async ({
+      page,
+      target,
+      pages: { relier, settings, signin },
+      testAccountTracker,
+    }) => {
+      const { email } = await signInAccount(
+        target,
+        page,
+        settings,
+        signin,
+        testAccountTracker
+      );
+
+      const query = new URLSearchParams({
+        login_hint: email,
+        return_on_error: 'false',
+      });
+      await page.goto(`${target.relierUrl}/?${query.toString()}`);
+      await relier.signInPromptNone();
+
+      //Verify logged in to relier
+      expect(await relier.isLoggedIn()).toBe(true);
+    });
+
+    test('succeeds if no login_hint is provided', async ({
+      page,
+      target,
+      pages: { relier, settings, signin },
+      testAccountTracker,
+    }) => {
+      await signInAccount(target, page, settings, signin, testAccountTracker);
+
+      const query = new URLSearchParams({
+        return_on_error: 'false',
+      });
+      await page.goto(`${target.relierUrl}/?${query.toString()}`);
+      await relier.signInPromptNone();
+
+      //Verify logged in to relier
+      expect(await relier.isLoggedIn()).toBe(true);
+    });
+
+    test('fails if login_hint is different to logged in user', async ({
+      page,
+      target,
+      pages: { relier, settings, signin },
+      testAccountTracker,
+    }) => {
+      const loginHintAccount = testAccountTracker.generateAccountDetails();
+      await signInAccount(target, page, settings, signin, testAccountTracker);
+
+      const query = new URLSearchParams({
+        login_hint: loginHintAccount.email,
+        return_on_error: 'false',
+      });
+      await page.goto(`${target.relierUrl}/?${query.toString()}`);
+      await relier.signInPromptNone();
+
+      //Verify error message
+      await expect(
+        page.getByText('A different user is signed in')
+      ).toBeVisible();
+    });
+  });
+});
+
+async function signInAccount(
+  target: BaseTarget,
+  page: Page,
+  settings: SettingsPage,
+  signin: SigninPage,
+  testAccountTracker: TestAccountTracker
+): Promise<Credentials> {
+  const credentials = await testAccountTracker.signUp();
+  await page.goto(
+    `${target.contentServerUrl}/?forceExperiment=generalizedReactApp&forceExperimentGroup=react`
```

Comment:
This is the only difference between these prompt none tests and the original.  However, locally I absolutely cannot get myself force enrolled into the experiment.  So maybe this is a bit pointless.  I did hard code a `return true` in `isInReactExperiment` and the functional tests did pass with the React version of `/authorization`.

Reply:
Word on the street is that https://github.com/mozilla/fxa/pull/18536 will fix force enrollment.  So we can leave this here.

Reply:
Thanks for transferring these over!